Solutions/GitLab/SolutionMetadata.json

```diff
@@ -1,35 +1,16 @@
 {
   "publisherId": "azuresentinel",
   "offerId": "azure-sentinel-solution-gitlab",
-  "version": "1.0.0",
-    "dependencies": {
-        "criteria": [
-            { "kind": "AnalyticsRule", "contentId": "2238d13a-cf05-4973-a83f-d12a25dbb153", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "c1544d8f-cbbd-4e35-8d32-5b9312279833", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "0f4f16a2-b464-4c10-9a42-993da3e15a40", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "e0b45487-5c79-482d-8ac0-695de8c031af", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "7241740a-5280-4b74-820a-862312d721a8", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "4d6d8b0e-6d9a-4857-a141-f5d89393cddb", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "8b291c3d-90ba-4ebf-af2c-0283192d430e", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "3efd09bd-a582-4410-b7ec-5ff21cfad7bd", "version": "1.0.0" },
-            { "kind": "AnalyticsRule", "contentId": "57b1634b-531d-4eab-a456-8b855887428f", "version": "1.0.0" },
-            { "kind": "DataConnector", "contentId": "GitLab", "version": "1.0.0" },
-            { "kind": "Parser", "contentId": "GitLabAccess", "version": "1.0.0" },
-            { "kind": "Parser", "contentId": "GitLabAudit", "version": "1.0.0" },
-            { "kind": "Parser", "contentId": "GitLabApp", "version": "1.0.0" }
-        ]
-    },
-    "firstPublishDate": "2022-03-13",
-    "lastPublishDate": "2022-03-13",
-    "providers": ["Cisco"],
-    "categories": {
-        "verticals": [],
+  "firstPublishDate": "2022-04-27",
+  "lastPublishDate": "2022-06-27",
+  "providers": ["Cisco"],
+   "categories": {
```

Comment:
Please verify the provider and category in `SolutionMetadata` file. I guess this need to be updated to below as per email sent by Darshana.

![image](https://user-images.githubusercontent.com/107389644/176107250-3c43fd94-5215-44aa-853b-9791646599ac.png)